src/commands/tsgen.ts

```diff
@@ -45,6 +45,14 @@ export default class TypeScriptCodeGeneratorCommand extends Command {
       default: true,
       allowNo: true,
     }),
+
+    branch: flags.string({
+      char: 'b',
```

Comment:
@vkalta Remove this short flag, current cli does not have any short flag for a branch flag.

Reply:
@vkalta Can you please check this?